Chap_API_Proc_Mgmt.tex

```diff
@@ -448,7 +464,7 @@ \subsection{Spawn attributes}
 }
 %
 \declareAttribute{PMIX_REPORT_BINDINGS}{"pmix.repbind"}{bool}{
-Report bindings of the individual processes.
+Report bindings of the individual processes.  How and where this information is reported is implementation dependent as well as dependent on whether the processes are created through a launching tool or by a direct call to \refapi{PMIx_Spawn}.
```

Comment:
Again, PMIx isn't doing anything here - we just pass the attribute to the host requesting that _it_ generate this report.